src/vms/context/context.ts

```diff
@@ -48,5 +49,9 @@ export const getContextFromURI = async (
     addSubnetDelegatorFee,
     networkID,
     hrp: getHRP(networkID),
+
+    // TODO: Populate these values once they are exposed by the API
+    gasPrice: 0n,
+    complexityWeights: createEmptyDimensions(),
```

Comment:
This will part of the next round of work.

Reply:
Yeah I was wondering the same thing recently. But since there is no work done yet in avalanchego (from what I've seen and I haven't gotten a response yet) that shows them actually setting up these values in the context, I was just going leave it here for now.

I suspect this may need to be adjusted once we start work to pull these values from the API, but I'm not exactly sure how that adjustment is going to look yet (the API to expose this doesn't appear to be in place yet to reference).